tests/test-cases/Group1-Docker-Commands/1-03-Docker-Images.robot

```diff
@@ -30,19 +30,19 @@ Simple images
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull alpine:3.1
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images alpine
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
     Should Contain X Times  ${output}  alpine  3
 
 All images
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images -a
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images -a alpine
```

Comment:
This is fundamentally changing the nature of this test

Reply:
Again, it doesn't.  This test assumes an independent VCH.  It pulls down 3 versions of Alpines and want to see if docker images -a returns 3 images.  Adding a name to docker images doesn't simply filters images by that name.